### PR TITLE
Added episode protocols

### PR DESCRIPTION
**Pull request by doctor3w**
Added a display view protocol to unify our cell setup for episode cells. Ignore the data source for now, it's not used just yet but more as an example for now. 

This is implemented on the BookmarkViewController.

### CODE REVIEW

**Review by jackthmp — DISMISSED** [review]
LGTM, are you planning on deleting `func setup` in `BookmarkTableViewCell`?

**Comment by doctor3w** [comment]
Only if BookmarkViewController is the only vc that uses it. But I have a feeling it isn't. Mindy and I's next job anyway is going through and putting this as many places as we can anyway and remove setup functions so I can just do it then.

File: Podcast/BookmarkViewController.swift [hunk 1]
```diff
@@ -159,10 +159,33 @@ extension BookmarkViewController: UITableViewDataSource {
...
+        cell.displayView.set(seriesTitle: episode.seriesTitle)
+        cell.displayView.set(title: episode.title)
+        if let url = episode.smallArtworkImageURL {
+            cell.displayView.set(smallImageUrl: url)
+        }
+        if let url = episode.largeArtworkImageURL {
+            cell.displayView.set(largeImageUrl: url)
+        }
+        
+        cell.displayView.set(topics: episode.topics.map { topic in topic.name })
```
Comment (AAAstorga) [inline Podcast/BookmarkViewController.swift:178]: nice use of map

File: Podcast/EpisodeProtocols.swift [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    func set(numberOfRecasts: Int)
+    
+    func set(downloadStatus: DownloadStatus)
+}
+
+class EpisodeTableDataSource: NSObject {
+    
+    // Can later adapt this to multiple cell types (like series and episodes)
+    private var sections: Int = 1
+    var identifier: String = "my_cell"
```
Comment (AAAstorga) [inline Podcast/EpisodeProtocols.swift]: can this be a better identifier

File: Podcast/EpisodeProtocols.swift [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    func set(recastBlurb: String)
+    func set(numberOfRecasts: Int)
+    
+    func set(downloadStatus: DownloadStatus)
+}
+
+class EpisodeTableDataSource: NSObject {
+    
+    // Can later adapt this to multiple cell types (like series and episodes)
+    private var sections: Int = 1
```
Comment (AAAstorga) [inline Podcast/EpisodeProtocols.swift]: don't need type definition here

File: Podcast/EpisodeProtocols.swift (hunk 2 above)
Comment (AAAstorga) [reply Podcast/EpisodeProtocols.swift]: also use type inference

**Review by AAAstorga — DISMISSED** [review]
I approve, but some minor style changes! Looks great. Love all the work being put in for this refactor!

File: Podcast/EpisodeProtocols.swift (hunk 2 above)
Comment (doctor3w) [reply Podcast/EpisodeProtocols.swift]: oh this isn't used for anything rn, I was just playing around to see how I could abstract this more. I just wanted a test identifier to write out the code. Same with section

**Pull request merged**